apps/wing-console/console/server/src/router/function.ts

```diff
@@ -46,7 +46,7 @@ export const createFunctionRouter = () => {
         try {
           const response: ResponseEnvelope = {
             success: true,
-            response: await client.invoke(input.message),
+            response: (await client.invoke(input.message)) ?? "",
```

Comment:
I think it'd make more sense to change the `ResponseEnvelope` to ```
export type ResponseEnvelope =
  | {
      success: true;
      response: string | undefined;
    }
  | {
      success: false;
      error: unknown;
    };
```

Reply:
Nice! Done.